pcf/test/particle/aws/iam/test_iam_role.py

```diff
@@ -46,7 +46,8 @@ def test_apply_states(self):
             "flavor":"iam_role", # Required
             "aws_resource":{
                 "custom_config": {
-                    "policy_arns": []
+                    "policy_arns": [],
+                    "IsInstanceProfile": True
```

Comment:
can you add an assert showing that the profile is created and destroyed will the role. otherwise looks good thanks

Reply:
added test but couldn't test terminate cuz moto doens't have delete_instance_profile() implemented yet.